src/content/other/hydroelectric.mdx

```diff
@@ -1,14 +1,16 @@
 ---
 title: Hydroelectric Power
 description: The power of water
-sectionHeader: Hypostatic? Nope, Just Hydroelectric
+sectionHeader: Hydroelectric Power
 ---
 
 {/* Is the Hypostatic Union joke too bad? */}
 
-Water is powerful.
-It generates a lot of power in Tennessee and Washington State.
+While not as green as other initiatives due to its environmental effects, hydropower is still a great source of electricity.
+Hydropower has been used in the US for nearly 100 years. It was a key part of New Deal legislation and new dams are still being built to this day.
+The biggest issue with hydropower though is its impact on local ecosystems. It is reported that dams may obstruct fish migration, cause floods, and more.
+Overall, hydropower is a great resource just not the one that will power a green future.
```

Comment:
The end sounds like something that could be modified to enlargen the nuclear power one.

```suggestion
While not as green as other initiatives due to its environmental effects, hydropower is still a great source of electricity.
Hydropower has been used in the US for nearly 100 years. It was a key part of FDR's New Deal legislation, and new dams are still being built to this day.
The biggest issue with hydropower is its impact on local ecosystems.
It is reported that dams obstruct fish migration, cause floods, and more.
Overall, hydropower is a great renewable resource while we're still migrating to renewables, just not one that will power an even cleaner and greener future.
```